src/main/kotlin/com/stefan_grafberger/streamdq/anomalydetection/model/AnomalyCheckResult.kt

```diff
@@ -0,0 +1,8 @@
+package com.stefan_grafberger.streamdq.anomalydetection.model
+
+data class AnomalyCheckResult(
+        val value: Double?,
+        val isAnomaly: Boolean? = null,
+        val confidence: Double,
```

Comment:
Can you comment a bit more on this confidence attribute please? How is it calculated? Is there only one clear way to calculate it or is it ambiguous? Does every anomaly detection strategy have a way to give meaningful confidence values? If not, we could consider either removing it or making it optional.

Reply:
This confidence is ambiguous, I made it optional

Reply:
I saw that almost every (or is it every?) anomaly check just always uses 1.0 for the confidence. Or did I miss something? If it's not used/or only used by a very small number of strategies, I think we  can remove the confidence completely, what do you think?